silk/static/silk/lib/url.js

```diff
@@ -0,0 +1,199 @@
+(function() {
```

Comment:
Is this an unminified copy of `url.min.js`?  Do we need both?

Reply:
Yes and no, I will remove it.

Reply:
@albertyw I removed this unnecessary file